Improve documentation about Nomad ACL minimum rights (#10482)
containous/traefik: docs/content/providers/nomad.md

_Optional, Default=""_

Token is used to provide a per-request ACL token, if Nomad ACLs are enabled.

> The ACL rights that you should apply to this token is 'read-job' as described here in the [nomad documentation about ACL](https://developer.hashicorp.com/nomad/tutorials/access-control/access-control-policies)

Reviewer: ```suggestion
The appropriate ACL privilege for this token is 'read-job', as outlined in the [Nomad documentation on ACL](https://developer.hashicorp.com/nomad/tutorials/access-control/access-control-policies).
```
